orchestra/templates/communication/available_staffing_requests.html

```diff
@@ -15,7 +15,7 @@
       <table class="table">
         <tr>
           <th class="row-project">Project</th>
-          <th class="row-project-description">Project description</th>
+          <th class="row-project-description tags-col">Project description</th>
```

Comment:
why does the header need the new class?

Reply:
To add extra right margin to labels. Note the css code moved to `orchestra/static/dist/orchestra/common/css/orchestra.css`